### PR TITLE
EDITORIAL CLOSING: XMPP Newsletter December 2022 & January 2023

### PR DESCRIPTION
**Pull request by Echolon**
## This is the draft for the next release covering December 2022 & January 2023.

Final release is planned for 5th of February 2023.

You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: [commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org)

[You can use our online Pad](https://pad.nixnet.services/oHnY_ZvLT8SoFyCqIC2ung)! Please be tidy and write something instead of dropping bare HTTP links pray

Just paste your prepared text in this thread below or in our XMPP channel (xmpp:[commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org)?join)

Please do not just drop bare links - rather try to make a short and decent formulated description (<5 lines) what is the main outcome of your source. You do not need to write in a hard, formal tone - be passionate why this motivates you or what you work on!

XMPP Newsletter Timeline

Status:

* [x] Collection of news
* [x] Review
* [ ] Blog post + Tiny Newsletter
* [ ] Social media
* [ ] Translations

### CODE REVIEW

File: content/blog/newsletter/2022/01/2023-02-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+If you are interested in becoming an XSF member, the [listings are open until this 19th February 2023](https://wiki.xmpp.org/web/Membership_Applications_Q1_2023)!
+
+### XSF fiscal hosting projects
+
+The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:
+
+- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
+- [Mellium Co-op](https://opencollective.com/mellium)
+
+### Google Summer of Code 2022
```
Comment (wurstsalat3000) [inline content/blog/newsletter/2022/01/2023-02-05-newsletter.md:38]:
```suggestion
### Google Summer of Code 2023
```

File: content/blog/newsletter/2022/01/2023-02-05-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+{{< figure src="/images/logos/GSoC_2022_Logo.png" caption="XSF and Google Summer of Code 2023" >}}
```
Comment (wurstsalat3000) [inline content/blog/newsletter/2022/01/2023-02-05-newsletter.md:40]: image URL needs to be updated

Comment (Echolon) [reply content/blog/newsletter/2022/01/2023-02-05-newsletter.md:40]: Can you help me to get the SVG in here correctly? :(

Comment (wurstsalat3000) [reply content/blog/newsletter/2022/01/2023-02-05-newsletter.md]: Done!

File: content/blog/2023-02-05-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+Snikket informs about the [F-Droid security warnings](https://snikket.org/blog/fdroid-security-warning/) and their [Security update](https://snikket.org/blog/fdroid-security-update/). The issues has been solved already.
+
+PGPainless: [Use Any SOP Binary With SOP-Java and External-SOP](https://blog.jabberhead.tk/2023/01/13/use-any-sop-binary-with-sop-java-and-external-sop/)
+
+From the ignite relatime community:
+
+- [Spark 3.0.1 Released](https://discourse.igniterealtime.org/t/spark-3-0-1-released/92353)
+- [Denial of Service Vulnerability in Smack 4.4 if XMPPTCPConnection is used with StAX](https://discourse.igniterealtime.org/t/denial-of-service-vulnerability-in-smack-4-4-if-xmpptcpconnection-is-used-with-stax/92314)
```
Comment (wurstsalat3000) [inline content/blog/2023-02-05-newsletter.md:74]:
```suggestion
- A Denial of Service Vulnerability in Smack 4.4 [has been fixed](https://discourse.igniterealtime.org/t/denial-of-service-vulnerability-in-smack-4-4-if-xmpptcpconnection-is-used-with-stax/92314)
```

File: content/blog/2023-02-05-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+- [New Mastodon service for the Ignite Realtime community!](https://discourse.igniterealtime.org/t/new-mastodon-service-for-the-ignite-realtime-community/92389)
```
Comment (wurstsalat3000) [inline content/blog/2023-02-05-newsletter.md:75]:
```suggestion
- [The Ignite Realtime community now offers a Mastodon service](https://discourse.igniterealtime.org/t/new-mastodon-service-for-the-ignite-realtime-community/92389)
```

File: content/blog/2023-02-05-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+- [Help us translate Spark and Openfire!](https://discourse.igniterealtime.org/t/help-us-translate-spark-and-openfire/92399)
```
Comment (wurstsalat3000) [inline content/blog/2023-02-05-newsletter.md:76]:
```suggestion
- [The community started a call for help with translating Spark and Openfire](https://discourse.igniterealtime.org/t/help-us-translate-spark-and-openfire/92399)
```

File: content/blog/2023-02-05-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+
+[EU NIS 2 directive: does it apply to anyone who provides free online digital services?](https://notes.nicfab.it/en/posts/nis2/)
```
Comment (wurstsalat3000) [inline content/blog/2023-02-05-newsletter.md:78]:
```suggestion
nicfab writes about the new [EU NIS 2 directive: does it apply to anyone who provides free online digital services?](https://notes.nicfab.it/en/posts/nis2/)
```

File: content/blog/2023-02-05-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+
+[Gajim 1.6.0](https://gajim.org/post/2023-01-07-gajim-1.6.0-released/) and [1.6.1](https://gajim.org/post/2023-01-14-gajim-1.6.1-released/) have been released. For Gajim 1.6, developers ported Gajim’s and python-nbxmpp’s underlying HTTP library to libsoup3. Also, audio previews now look nicer and allow for more control (playback speed, jumping). Emoji rendering on MacOS has been repaired and many fixes have been implemented under the hood.
+
+Libervia has been granted a [NLnet NGI Assure Fund for implementation of Audio/Video Calls](https://nlnet.nl/project/Libervia-AV/).
+
+[Moxxy 0.4.0](https://codeberg.org/moxxy/moxxy/releases/tag/v0.4.0) and [0.4.1](https://codeberg.org/moxxy/moxxy/releases/tag/v0.4.1) have been released. These two releases bring stickers, reactions, voice message recording and various improvements and fixes to the UI and XMPP code.
+
+[Spark 3.0.1 has been released](https://github.com/igniterealtime/Spark/releases), which brings some bug fixes
```
Comment (wurstsalat3000) [inline content/blog/2023-02-05-newsletter.md:93]:
```suggestion
[Spark 3.0.1 has been released](https://github.com/igniterealtime/Spark/releases), which brings some bug fixes.
```

File: content/blog/2023-02-05-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+
+### Servers
+
+[ejabberd 23.01 has been released](https://www.process-one.net/blog/ejabberd-23-01/) Almost three months after the previous release, ejabberd 23.01 includes many bug fixes, several improvements and some new features. A new module, mod_mqtt_bridge, can be used to replicate changes to MQTT topics between local and remote servers. Supported XEPs are now automatically gathered from source code documentation in order to provide a DOAP file for further processing on [xmpp.org/servers/ejabberd](https://xmpp.org/software/servers/ejabberd/).
```
Comment (wurstsalat3000) [inline content/blog/2023-02-05-newsletter.md:97]:
```suggestion
[ejabberd 23.01 has been released](https://www.process-one.net/blog/ejabberd-23-01/). Almost three months after the previous release, ejabberd 23.01 includes many bug fixes, several improvements and some new features. A new module, mod_mqtt_bridge, can be used to replicate changes to MQTT topics between local and remote servers. Supported XEPs are now automatically gathered from source code documentation in order to provide a DOAP file for further processing on [xmpp.org/servers/ejabberd](https://xmpp.org/software/servers/ejabberd/).
```

File: content/blog/2023-02-05-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+
+[Prosody 0.12.2 has been released](https://blog.prosody.im/prosody-0.12.2-released/) and [Prosody 0.11.14 released](https://blog.prosody.im/prosody-0.11.14-released/), which contains allowance U+7F when constructing stanzas, bug fixes and improvements.
```
Comment (wurstsalat3000) [inline content/blog/2023-02-05-newsletter.md:99]:
```suggestion
[Prosody 0.12.2](https://blog.prosody.im/prosody-0.12.2-released/) and [Prosody 0.11.14](https://blog.prosody.im/prosody-0.11.14-released/) have been released.  The 0.12 release brings bug fixes and improvements.
```

File: content/blog/2023-02-05-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+
+[Tigase XMPP Server 8.3 has been released](https://tigase.org/blog/tigase-xmpp-server-8.3.0/) -  it includes Tigase MEET - Tigase's take at SFU (Selective Forwarding Unit) group video-calls, support for XEP-0440 SASL Channel Binding Type Capability as well as initial, preview support for SASL2 and Bind2 and lots of usual fixes.
```
Comment (wurstsalat3000) [inline content/blog/2023-02-05-newsletter.md:101]:
```suggestion
[Tigase XMPP Server 8.3 has been released](https://tigase.org/blog/tigase-xmpp-server-8.3.0/). Tt includes Tigase MEET - Tigase's take at SFU (Selective Forwarding Unit) group video-calls, support for XEP-0440 SASL Channel Binding Type Capability as well as initial, preview support for SASL2 and Bind2 and lots of usual fixes.
```

File: content/blog/2023-02-05-newsletter.md [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+
+## Extensions and specifications
+
+Developers and other standards experts from around the world collaborate on these extensions, developing new specifications for emerging practices, and refining existing ways of doing things. Proposed by anybody, the particularly successful ones end up as Final or Active - depending on their type - while others are carefully archived as Deferred. This life cycle is described in [XEP-0001](https://xmpp.org/extensions/xep-0001.html), which contains the formal and canonical definitions for the types, states, and processes. [Read more about the standards process](https://xmpp.org/about/standards-process.html). Communication around Standards and Extensions happens in the [Standards Mailing List](https://mail.jabber.org/mailman/listinfo/standards) ([online archive](https://mail.jabber.org/pipermail/standards/)).
```
Comment (wurstsalat3000) [inline content/blog/2023-02-05-newsletter.md:119]:
```suggestion
The XMPP Standards Foundation develops extensions to XMPP in its [XEP series](https://xmpp.org/extensions/) in addition to [XMPP RFCs](https://xmpp.org/rfcs/).

Developers and other standards experts from around the world collaborate on these extensions, developing new specifications for emerging practices, and refining existing ways of doing things. Proposed by anybody, the particularly successful ones end up as Final or Active - depending on their type - while others are carefully archived as Deferred. This life cycle is described in [XEP-0001](https://xmpp.org/extensions/xep-0001.html), which contains the formal and canonical definitions for the types, states, and processes. [Read more about the standards process](https://xmpp.org/about/standards-process.html). Communication around Standards and Extensions happens in the [Standards Mailing List](https://mail.jabber.org/mailman/listinfo/standards) ([online archive](https://mail.jabber.org/pipermail/standards/)).
```

File: content/blog/2023-02-05-newsletter.md [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+
+xmpp.org features [a page about XMPP RFCs](https://xmpp.org/rfcs/) as well.
```
Comment (wurstsalat3000) [inline content/blog/2023-02-05-newsletter.md:121]:
```suggestion
```

File: content/blog/2023-02-05-newsletter.md [hunk 13]
```diff
@@ -0,0 +1,224 @@
...
+
+{{< newsletter-subscribe >}}
+
+Also check out our [RSS Feed](https://xmpp.org/feeds/all.atom.xml)!
+
+Looking for job offers or want to hire a professional consultant for your XMPP project? Visit our [XMPP job board](https://xmpp.work/).
+
+## Help us to build the newsletter
+
+This XMPP Newsletter is produced collaboratively by the XMPP community. Each month's newsletter issue is drafted in this [simple pad](https://yopad.eu/p/xmpp-newsletter-365days). At the end of each month, the pad's content is merged into the [XSF Github repository](https://github.com/xsf/xmpp.org/milestone/3). We are always happy to welcome contributors. Do not hesitate to join the discussion in our [Comm-Team group chat (MUC)](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort. You have a project and want to spread the news? Please consider sharing your news or events here, and promote it to a large audience.
```
Comment (wurstsalat3000) [inline content/blog/2023-02-05-newsletter.md:210]:
```suggestion
This XMPP Newsletter is produced collaboratively by the XMPP community. Each month's newsletter issue is drafted in this [simple pad](https://pad.nixnet.services/oHnY_ZvLT8SoFyCqIC2ung). At the end of each month, the pad's content is merged into the [XSF Github repository](https://github.com/xsf/xmpp.org/milestone/3). We are always happy to welcome contributors. Do not hesitate to join the discussion in our [Comm-Team group chat (MUC)](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort. You have a project and want to spread the news? Please consider sharing your news or events here, and promote it to a large audience.
```

File: content/blog/2023-02-05-newsletter.md [hunk 14]
```diff
@@ -0,0 +1,224 @@
...
+date: 2023-02-05
+categories: ['Newsletter']
+---
+
+Welcome to the XMPP Newsletter, great to have you here again! This issue covers the month of December 2022 & January 2023. 
+This is the first release of this year and we have after a well-deserved winter break! Many thanks to all our readers and all contributors!
+
+Like this newsletter, many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, especially throughout the current situation, please consider saying thanks or help these projects! Interested in supporting the Newsletter team? Read more at the bottom.
+
+### Newsletter Contributors & Translations
```
[the remark this answers is not in the document]
Comment (Echolon) [reply content/blog/2023-02-05-newsletter.md:12]: Ok

File: content/blog/2023-02-05-newsletter.md [hunk 15]
```diff
@@ -0,0 +1,224 @@
...
+
+[JMP's December Newsletter](https://blog.jmp.chat/b/december-newsletter-2022) talks about their new co-op structure, an important security fix in the Cheogram Android app, and a review of the highlights of 2022.  [JMP's January Newsletter](https://blog.jmp.chat/b/january-newsletter-2023) mentions another new release including a new [XMPP Threaded Chat UI](https://kumi.tube/w/1LQQp5Uia4u8Pdojxen1y8) and new payment options for the service. Last but not least: [Writing a Chat Client from Scratch](https://blog.jmp.chat/b/2022-chat-client-from-scratch) in Ruby.
+
+Snikket informs about the [F-Droid security warnings](https://snikket.org/blog/fdroid-security-warning/) and their [Security update](https://snikket.org/blog/fdroid-security-update/). The issues has been solved already.
+
+PGPainless: [Use Any SOP Binary With SOP-Java and External-SOP](https://blog.jabberhead.tk/2023/01/13/use-any-sop-binary-with-sop-java-and-external-sop/)
+
+From the ignite relatime community:
+
+- [Spark 3.0.1 Released](https://discourse.igniterealtime.org/t/spark-3-0-1-released/92353)
```
[the remark this answers is not in the document]
Comment (Echolon) [reply content/blog/2023-02-05-newsletter.md:73]: Let's keep it anyway

**Comment by Echolon** [comment]
@wurstsalat3000 thanks for all the quality!

**Comment by Echolon** [comment]
## Editorial closing of the XMPP Newsletter December '22 and January '23 release

This is the last release for this year! Many many thanks to everyone for their help in creation, review and translation!!!

- English (original):  **Alexander "PapaTutuWawa", emus, Licaon_Kter, Ludovic Bocquet, MattJ, MSavoritias (fae,ve), wurstsalat, Zash** 
- French: **Adrien Bourmault (neox), alkino, anubis, Benoît Sibaud, Pierre Jarillon, Ppjet6, Ysabeau**
- German: **Jeybe, wh0nix**
- Italian: **nicfab**
- Spanish: **daimonduff, TheCoffeMaker**

- :heart: :fireworks:  Next release will be next year on **5th of March 2023**!

Have a good time until then and see you soon!

**Pull request merged**